benchmarks/Makefile

```diff
@@ -1,17 +1,34 @@
+# Get the current ISO8601 date time format but remove + and :
+$(shell date -I'minutes'| cut -d+ -f1 | sed 's/:/-/'> ./current)
```

Comment:
I don't like how I've done this, but I couldn't get the variable to store in Make without this re-executing every time. I'm relatively new to make... in that this is the 2nd make file I've ever contributed to.